components/prism-cil.js

```diff
@@ -0,0 +1,27 @@
+Prism.languages.cil = {
+	'comment': /\/\/.*/,
+	
+	'string': {
+		pattern: /(?:["'])(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+		greedy: true
+	},
+	
+	'directive': {
+		pattern: /([^a-zA-Z0-9])\.[a-z]+\b/,
```

Comment:
1. `.method` will not be matched. Please change the first group to `(^|\W)` (`[^a-zA-Z0-9]` == `\W`).
2. `.ctor()` will be matched. Isn't there always at least one space after a directive, so why don't we use `(?=\s)` instead of `\b`?

Reply:
Good thinking! I didn't know JS supported positive lookaheads.